Sources/Sentry/Public/SentryOptions.h

```diff
@@ -707,6 +707,20 @@ NS_SWIFT_NAME(Options)
  */
 @property (nullable, nonatomic, copy) SentryBeforeEmitMetricCallback beforeEmitMetric;
 
+/**
+ * Enables the User Feedback integration. See `configureUserFeedback` for options that customize the
+ * experience.
+ * @warning This is an experimental feature and may still have bugs.
+ * @note Default value is @c NO .
+ */
+@property (nonatomic, assign) BOOL enableUserFeedbackIntegration;
```

Comment:
Is this enabling the widget? if so, should we call it:
```suggestion
@property (nonatomic, assign) BOOL enableUserFeedbackWidget;
```

Reply:
The integration manages the widget, but looking at the other properties in options that manage enabling integrations, we don't actually put "Integration" in the name, so I'd _would_ be ok just naming it enableUserFeedback, as long as we don't think there's confusion with the preexisting API as you mentioned above (which is why I tacked on the "Integration" here)

Reply:
That's fair

Reply:
LOL I can tell it was useless because I hadn't even set it in the sample apps 😆